nixos/modules/programs/firefox.nix

```diff
@@ -287,7 +287,7 @@ in
         (_: value: { Value = value; Status = cfg.preferencesStatus; })
         cfg.preferences);
       ExtensionSettings = builtins.listToAttrs (builtins.map
-        (lang: builtins.nameValuePair
+        (lang: lib.attrsets.nameValuePair
```

Comment:
This should be just lib.nameValuePair

Reply:
Should I open another PR?

Should I also add this to tests as 
```diff
diff --git a/nixos/tests/firefox.nix b/nixos/tests/firefox.nix
index 6418e029f80d..bd479d081f3b 100644
--- a/nixos/tests/firefox.nix
+++ b/nixos/tests/firefox.nix
@@ -16,6 +16,7 @@ import ./make-test-python.nix ({ lib, pkgs, firefoxPackage, ... }:
         enable = true;
         preferences."media.autoplay.default" = 0;
         package = firefoxPackage;
+        languagePacks = [ "de" ];
       };

       # Create a virtual sound device, with mixing

Reply:
You don't need to, i just wanted to leave that here a note for next time. 

The organisation inside of lib is mostly internal and usage of it don't need to account for it.